backend/server/minimal_demo.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	// Advisors.
 	_ "github.com/bytebase/bytebase/backend/plugin/advisor/mysql"
+	_ "github.com/bytebase/bytebase/backend/plugin/advisor/oceanbase"
```

Comment:
No need to add Oceanbase advisor in minimal mode.

Reply:
This advisor is the only one under the 'oceanbase' module, and this module has not been referenced by other modules. I found other modules are basically referenced in minimal or minimal_demo, so I added it here. Should I move it to anywhere else?

Reply:
The `minimal` mode is only used for some dev environments. For prod environments, we build the bytebase without the minimal tag. So you can register the `oceanbase` module in `backend/server/ultimate.go`